activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -112,7 +112,7 @@ def missing_default_forged_as_empty_string?(default)
 
       NATIVE_DATABASE_TYPES = {
         :primary_key              => "int auto_increment PRIMARY KEY",
-        :primary_key_no_increment => "int(11) PRIMARY KEY", #RingRevenue patch
+        :primary_key_no_increment => "int(11) PRIMARY KEY", # Invoca patch
```

Comment:
Scott had some feedback about preferring a different format for patch comments. I would use his suggested comment pattern here and in the rest of this PR unless he has given the thumbs up to do otherwise